content/departments/product-engineering/engineering/cloud/devops/managed/cost_estimation.md

```diff
@@ -64,4 +65,24 @@ The following summarizes the infrastructure costs incurred for a managed instanc
 | $1.42/GB                       | [Cloud Proxy](https://cloud.google.com/vpc/network-pricing#proxy-instance-charge), 3 are required.                                                                                                                                |
 | $0.12/GB                       | [Internet egress traffic](https://cloud.google.com/vpc/network-pricing#internet_egress) from `us-central1`.                                                                                                                       |
 
+### Cloud SQL usage
+
+Cloud SQL sizing is depending on number of repos
+
+1-500 repos
+
+`db-custom-4-6144`, 100G, $180.26/mo
+
+500-2000 repos
+
+`db-custom-4-10240`, 100G, $200.70/mo
+
+2000+ repos
+
+`db-custom-4-12288`, 100G, $210.92/mo
```

Comment:
this is really just unscientific guess, we don't have any real-world data on Cloud SQL yet.